### PR TITLE
Mobile success stories

### PR DESCRIPTION
**Pull request by codyloyd**

### CODE REVIEW

**Comment by codyloyd** [comment]
every stinkin time

**Comment by codyloyd** [comment]
eh.. something weird happened to the indentation. Thanks VIM.

fixing now

**Comment by KevinMulhern** [comment]
👍 LGTM

File: app/assets/stylesheets/components/success_stories.scss [hunk 1]
```diff
@@ -20,67 +23,100 @@
...
- }
+  }
 
   .success-stories-read-more {
     width: 100%;
     text-align: center;
-
+    padding-top: 30px;
     a {
       color: rgba(74, 74, 74, 0.7);
```
Comment (arun1595) [inline app/assets/stylesheets/components/success_stories.scss]: You didn't modify this line but can you please replace the color with the variable `$grey`?

File: app/assets/stylesheets/components/success_stories.scss [hunk 2]
```diff
@@ -20,67 +23,100 @@
...
-    font-weight: light;
+    font-weight: 100;
     width: 100%;
+    padding: 0 40px;
     text-align: center;
     letter-spacing: 0.4px;
-    color: rgba(74, 74, 74, 0.7);
-
+    p {
+      color: rgba(74, 74, 74, 0.7);
```
Comment (arun1595) [inline app/assets/stylesheets/components/success_stories.scss]: Ditto. Use the variable `$grey`

**Pull request merged**